src/nimblepkg/nimscriptwrapper.nim

```diff
@@ -119,7 +119,13 @@ proc execScript(scriptName, actionName: string, options: Options):
     (output, exitCode) = execNimscript(nimsFile, scriptName.parentDir(), actionName, options)
 
   if exitCode != 0:
-    raise newException(NimbleError, output)
+    let
+      errOut =
+        if output.len != 0:
+          output
+        else:
+          "Exception raised during nimble script execution"
```

Comment:
IMO this would be easier to read:

```nim
var errMsg = output
if errMsg.len == 0:
  errMsg = "Exception raised during Nimble script execution"
```

Reply:
Alternatively, even just removing the extra indentation after the `let` would be better.